src/app/submissions/submissions.service.ts

```diff
@@ -316,7 +316,7 @@ export class SubmissionsService {
         });
         this.csvService.exportCSV({
           data,
-          title: `${toProperCase(type)} -  ${exam.name}${exam.description ? '\n' + exam.description : ''}`,
+          title:`Survey-${exam.name}`,
```

Comment:
Exports can be either exams or surveys, we need to differentiate, so we need: `${toProperCase(type)}`